### ISSUE TITLE
unable to write orientation to file

### ISSUE THREAD

**Issue by jose1711** [BestImageViewer/geeqie#578]
### ISSUE TYPE
* Bug Report

### GEEQIE VERSION
```
Geeqie 1.4
```

### OS / DISTRIBUTION
Arch Linux 64bit

### SUMMARY
```Edit - Orientation - Write orientation to file``` will result in a failure if a file contains brackets "("/")", possibly other non-alphanumeric characters too.

### STEPS TO REPRODUCE
Rename a jpeg file to ```image(1).jpg```, change orientation using "[" key, click ```Edit - Orientation - Write orientation to file``` - Observe an error.
CLI output:
```
sh: -c: line 0: syntax error near unexpected token `('
sh: -c: line 0: `/usr/lib/geeqie/geeqie-rotate -r 3 /tmp/image(1).jpg'
```

**Comment by jose1711** [comment]
the operation will fail also if a file is inside a directory containing space (e. g. /tmp/my photos)

**Comment by caclark** [comment]
Fixed in commit 4df045ff at geeqie.org
